node-graph/gstd/src/image_color_palette.rs

```diff
@@ -0,0 +1,20 @@
+use graphene_core::raster::ImageFrame;
+use graphene_core::Color;
+use graphene_core::Node;
+use log::debug;
+
+#[derive(Debug, Clone, Copy)]
+pub struct ImageColorPaletteNode {}
+
+#[node_macro::node_fn(ImageColorPaletteNode)]
+fn image_color_palette(image: ImageFrame<Color>) -> Vec<Color> {
+	const BLACK: Color = Color::from_rgbf32_unchecked(0., 0., 0.);
+	const WHITE: Color = Color::from_rgbf32_unchecked(1., 1., 1.);
+	const RED: Color = Color::from_rgbf32_unchecked(1., 0., 0.);
+	const GREEN: Color = Color::from_rgbf32_unchecked(0., 1., 0.);
+	const BLUE: Color = Color::from_rgbf32_unchecked(0., 0., 1.);
+
+	debug!("image_color_palette from debug: {:?}", image);
```

Comment:
I guess I missed something here because I could not reach this code here? How could I check this output on the UI? 

I have tried all the options on the "Help" menu.